src/Compilers/Test/Core/Assert/AssertEx.cs

```diff
@@ -656,7 +656,12 @@ public static string GetAssertMessage<T>(IEnumerable<T> expected, IEnumerable<T>
                 }
                 else
                 {
-                    itemInspector = new Func<T, string>(obj => (obj != null) ? obj.ToString() : "<null>");
+                    itemInspector = static obj => obj switch
+                    {
+                        null => "<null>",
+                        string s => $@"""{s}""",
```

Comment:
This makes it easier to paste baselines in from an AssertEx.Equal failure, since the string values will already be quoted.

I didn't bother to do escaping here, if anything I would like to output a `"""`-string only if the string is detected to contain any characters that need escaping. It's something which can be addressed when we hit it.

Reply:
This introduces a problem where every line of a VerifyIL baseline is quoted. I'll have to look at either adjusting that or reverting this change.